### PR TITLE
Python Docstring relocation

### PR DESCRIPTION
**Pull request by bheliker**
See Issue 417

### CODE REVIEW

**Comment by bheliker** [comment]
@googlebot I signed it!

**Comment by RiverMersey** [comment]
Oh, I really wish I understood what was going on here!
All that I do know is that @bheliker seems to be arguing with a software robot!

**Comment by bheliker** [comment]
I didn't have a signed CLA, so I followed the google bot instructions, which include that command, with the exclamation (I know, seems like I'm arguing with it, hah)
> Once you've signed (or fixed any issues), please reply here with @googlebot I signed it! and we'll verify it.

File: python/openlocationcode/openlocationcode.py [hunk 1]
```diff
@@ -228,25 +228,27 @@ def isFull(code):
...
-       range -90 to 90.
-   longitude: A longitude in signed decimal degrees. Will be normalised to
-       the range -180 to 180.
-   codeLength: The number of significant digits in the output code, not
-       including any separator characters.
-"""
 
 
 def encode(latitude, longitude, codeLength=PAIR_CODE_LENGTH_):
+
```
Comment (bilst) [inline python/openlocationcode/openlocationcode.py:234]: NIT: Looks like an extra newline snuck in here.

File: python/openlocationcode/openlocationcode.py [hunk 2]
```diff
@@ -379,37 +382,38 @@ def decode(code):
...
-       find the nearest matching full code.
-   referenceLongitude: The longitude (in signed decimal degrees) to use
-       to find the nearest matching full code.
- Returns:
-   The nearest full Open Location Code to the reference location that matches
-   the short code. If the passed code was not a valid short code, but was a
-   valid full code, it is returned with proper capitalization but otherwise
-   unchanged.
-"""
+
```
Comment (bilst) [inline python/openlocationcode/openlocationcode.py:385]: Please remove extra newline.

**Review by bilst — COMMENTED** [review]
Thanks for the fix! It looks like at some point autoformatting added padding between the out-of-place docstrings and function defs, so now there are four empty lines. Mind removing the padding too so that there will be two empty lines separating each top level function def?

**Pull request closed without merging**